tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.TestRuns/SmokeTests/MetricCountInt.cs

```diff
@@ -0,0 +1,31 @@
+using System.Runtime.CompilerServices;
+using Samples.Probes.TestRuns.Shared;
+
+namespace Samples.Probes.TestRuns.SmokeTests
+{
+    public class MetricCountInt : IRun
```

Comment:
I can only see tests for `metric count`, do you think it's worthwhile to also add tests for the other types of metrics?

Reply:
Fixed in 1b2566d1a442025260e7614b6c0f95f94133d32f